webhook/handlers/patient_first_workflow.js

```diff
@@ -0,0 +1,71 @@
+const db = require('../../models')
+
+class PatientFirstController {
+
+  async bookConsulation(agent, id){
+    let fulfillmentText = "";
+    let user = await db.userWA.findOne({where: {wa_phone_number: id}});
+    let patients = await db.patientWA.findAll({where: { user_id : user.id }});
```

Comment:
Change `user_id` to `User_id`

Reply:
https://github.com/kj7kunal/E-Med/blob/develop-patient-first-workflow/webhook/handlers/user_registration_workflow.js#L189